src/mobile/.env.development

```diff
@@ -0,0 +1,11 @@
+################################################################################
+# Supabase Services
+################################################################################
+
+# We 10.0.2.2 to access your actual machine inspire by https://stackoverflow.com/a/6310592
+SUPABASE_URL=http://10.0.2.2./v1/supabase
```

Comment:
This URL won't work for most devs.

Reply:
@humphd did you have a look at the `stackoverflow` comment above that line ?

Reply:
@humphd, I think I will replace this URL with our staging supabase  URL `https://dev.api.telescope.cdot.systems/v1/supabase`.
Once this PR is in and the  `quotes` table is created, we can manually insert all quotes in the supabase console.

Reply:
Perfect